Examples/Algorithms/Generators/ActsExamples/Generators/EventGenerator.cpp

```diff
@@ -43,7 +43,7 @@ std::string ActsExamples::EventGenerator::name() const {
 
 std::pair<std::size_t, std::size_t>
 ActsExamples::EventGenerator::availableEvents() const {
-  return {0u, SIZE_MAX};
+  return {0u, std::numeric_limits<std::size_t>::max()};
```

Comment:
This could maybe use a named sentinel value, instead of just an implicit one.

Reply:
I am not sure if we need a sentinel here, since this more or less defines the limits of the machine.
Or maybe a sentinel for the `pair` itself? because `0` is also just container limit.

Reply:
We converged to that it could be nice to have but also not really clear if we actually want it, since we don't really need it now.